app/controllers/pair_requests_controller.rb

```diff
@@ -27,8 +23,14 @@ def create
           redirect_to pair_request_url(@pair_request),
             notice: 'Pair request was successfully created.'
         end
+        format.turbo_stream do
+          @new_pair_request = @pair_request
+          @pair_request = PairRequest.new
```

Comment:
I think it should be `@new_pair_request = PairRequest.new` and `@pair_request` for what is saved.

Reply:
so I realized that this was actually intentional, though I probably could do a better job with naming conventions. By allowing `@pair_request` to stay the same, it lets us keep support the `@pair_request` errors that get shown without having to worry about two cases. I will update the name, however, to better reflect the status of the persisted pair request.

Reply:
#102 should address this issue